src/frontends/tensorflow_lite/src/input_model.cpp

```diff
@@ -162,6 +170,17 @@ void InputModel::InputModelTFLiteImpl::load_model() {
             m_telemetry->send_event("op_count", "tflite_" + op.first, static_cast<int>(op.second));
         }
     }
+
+    size_t subgraph_size = m_graph_iterator->get_subgraph_size();
+    if (subgraph_size > 1) {
+        m_subgraphs.reserve(subgraph_size);
+        m_subgraphs.push_back(nullptr);  // no main graph
+        for (size_t i = 1; i < subgraph_size; ++i) {
+            m_subgraphs.push_back(
+                std::make_shared<ov::frontend::tensorflow_lite::InputModel>(m_graph_iterator->get_subgraph(i),
+                                                                            m_telemetry));
+        }
+    }
```

Comment:
I didn't understand. On the on hand, above it is said that sub-graph is created on demand. On the other hand, you create `InputModel` for all sub-graphs

Reply:
Commented in previous comment about the logic of this PR.